colosseum/validators.py

```diff
@@ -97,9 +100,66 @@ def is_color(value):
     try:
         value = parser.color(value)
     except ValueError as error:
-        raise ValidationError(str(error))
+        raise exceptions.ValidationError(str(error))
 
     return value
 
 
 is_color.description = '<color>'
+
+
+# https://www.w3.org/TR/2011/REC-CSS2-20110607/syndata.html#value-def-identifier
+_CSS_IDENTIFIER_RE = re.compile(r'^[a-zA-Z][a-zA-Z0-9\-\_]+$')
+
+
+def is_font_family(value):
+    """
+    Validate that value is a valid font family.
+
+    This validator returns a list.
+    """
+    from .constants import GENERIC_FAMILY_FONTS as generic_family
```

Comment:
Circular imports?

Reply:
Ah - right. That's often an indicator of a bigger problem, but in this case, I'm not sure there's going to be a cleaner way to separate things so that we don't have the circular import problem.

Reply:
We could split choices and constants so that choices holds the Choices logic plus the construction of different choices items and leave constants to only hold constants?

This might now get rid of all issues, but might better decouple the code base.